docs/manual/How to contribute to the development of the tool.rst

```diff
@@ -0,0 +1,27 @@
+=======================================================

```

Comment:
Also remove the old reference to contributing - e.g. see https://osemosys.readthedocs.io/en/latest/manual/Advanced%20functionalities.html#modifications-enhancements-and-extensions-of-the-main-code